src/components/Homepage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
```

Comment:
I am seeing it like this:
![image](https://user-images.githubusercontent.com/48275618/75627139-6bec5200-5bd6-11ea-9c9c-479aa61ec6fa.png)